test/utils/format.js

```diff
@@ -148,3 +149,10 @@ test('collectionHrefFromNameVersion returns a nullValue collection if the collec
 test('collectionHrefFromNameVersion returns a nullValue collection if the collection is undefined', function (t) {
   t.is('--', collectionHrefFromNameVersion());
 });
+
+test('fullDate returns the properly formatted date for midnight', function (t) {
+  const date = '2025-04-13T04:00:00Z'; //adjusted for UTC
```

Comment:
Z is for UTC, the comment is not necessary.
Can you update to a more specific time like '2025-04-13T04:15:32.567Z'?